public/favicon/site.webmanifest

```diff
@@ -3,14 +3,14 @@
   "short_name": "",
```

Comment:
**question:** @TenzDelek 
name and short_name are empty. Should I put there some info? For example,

"name": "DearDiary",
 "short_name": "DearDiary"?